OsmAnd-java/src/main/java/net/osmand/binary/RouteDataObject.java

```diff
@@ -341,13 +341,13 @@ public String getDestinationRef(boolean direction) {
 			}
 			//return names.get(region.refTypeRule);
 		}
-		return null;
+		return getRef(lang, transliterate, direction);
 	}
 
 	public String getDestinationName(String lang, boolean transliterate, boolean direction) {
-		//Issue #3289: Treat destination:ref like a destination, not like a ref
+		/*//Issue #3289: Treat destination:ref like a destination, not like a ref
```

Comment:
code needs to be deleted and not commented

Reply:
Ok